components/finance/FinanceCategoriesList/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import FinanceCategoriesListWrapper from '../FinanceCategoriesListWrapper'
```

Comment:
do not need a wrapper component if it does not have any specific responsibility

Reply:
fixed.